setup.py

```diff
@@ -55,6 +55,8 @@
                         'h5py >= 2.3.1', 'biom-format',
                         # forcing 0.1.4 because Qiita uses this version
                         'woltka @ https://github.com/'
-                        'qiyunzhu/woltka/archive/refs/tags/v0.1.4.zip'],
+                        'qiyunzhu/woltka/archive/refs/tags/v0.1.4.zip',
+                        'pysyndna @ https://github.com/AmandaBirmingham/'
```

Comment:
Is it all right to leave the reference pointing to Amanda's repo? Or should pysyndna be forked into qiita-spots?

Reply:
If we fork, we will fork to biocore; however, we have no idea when or if this will happen.